src/pages/community-champions.js

```diff
@@ -353,6 +359,90 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    color: ${({ theme }) => theme.colorMap.greyDarkOne};
```

Comment:
```suggestion
    border-color: ${({ theme }) => theme.colorMap.greyDarkOne};
```

This actually is done with a border color! `hr`s can be odd

Reply:
thanks for catching that! can't believe i didn't notice the line was not grey 🥲🥲

Reply:
👀